security/index.adoc

```diff
@@ -0,0 +1,90 @@
+[id="security-compliance-overview"]
+= {product-title} security and compliance
+include::modules/common-attributes.adoc[]
+:context: security-compliance-overview
+
+toc::[]
+
+// TODO: Pull out things that don't apply for previous versions
+
+// TODO: Need some help with an intro blurb
+
+[id="security-overview"]
+== Security overview
+
+It is important to understand how to properly secure various aspects of your {product-title} cluster.
```

Comment:
I might move this first sentence to the Container security section: "It is important to understand how to properly secure various aspects of your OpenShift Continer Platform cluster" since there is only one sentence in the Security overview section.

Reply:
@TanyaShearonRH I know that this could use beefed up (and also need an intro blurb at the top of the assembly). But usually I try to avoid having back-to-back headings with no text underneath a heading.